README.md

```diff
@@ -50,21 +50,33 @@ Run `echo $key && echo $token`.
 
 #### Create Moment JSON
 
-Run `make key=$key token=$token board=XXX moment=XXX import`.
+Run `make key=$key token=$token board=BOARD_NAME moment=MOMENT_NAME import` [-B to overwrite old version]
 
 The json should be created inside a directory which will be named after the Trello board's ID.
 
 #### Create Thumbnails
 
-Run `make key=$key token=$token board=XXX moment=XXX thumbnails`
+Run `make key=$key token=$token board=BOARD_NAME moment=MOMENT_NAME thumbnails`
 
 #### Create .TTL File
 
-Run `make key=$key token=$token board=XXX moment=XXX jop.ttl`
+Run `make key=$key token=$token board=BOARD_NAME moment=MOMENT_NAME [BOARD_NAME].ttl`
+
+### Create the Moment Descripton .TTL File
+
+Run `make key=$key token=$token board=BOARD_NAME moment=MOMENT_NAME [MOMENT_NAME]_moment.ttl`
+
+### Create the Moment .TTL File
+
+Run `make key=$key token=$token board=BOARD_NAME moment=MOMENT_NAME [MOMENT_NAME].ttl`
+
+#### Create .JSON File
+
+Run `make key=$key token=$token board=BOARD_NAME moment=MOMENT_NAME [MOMENT_NAME].json`
 
 #### Trello2Moment
 
-Run `node trello2moment --board=BOARD_NAME`
+Run `node trello2moment --board=BOARD_NAME --moment=MOMENT_NAME [--description=true]`
 
 ## LDP
```

Comment:
Some of this info can go into the `POD` in the Makefile now.

Reply:
I simplified the ReadMe and I'm currently working on moving it into the POD now.